src/main/java/com/team766/controllers/CanSparkMaxSmartMotionRotationalPIDController.java

```diff
@@ -0,0 +1,34 @@
+import java.*;
+
+public class CanSparkMaxSmartMotionRotationalPIDController extends ExtendableCanSparkMaxSmartMotionPIDController {
+	
+	private final double degreesToEncoderUnitsRatio;
+	private final double antiGravK;
+
+	public CanSparkMaxSmartMotionRotationalPIDController(final String configName, final double absEncoderOffset, final double absEncoderOffsetForZeroEncoderUnits, final OffsetPoint first, final OffsetPoint second) {
+		super(configName, absEncoderOffset, absEncoderOffsetForZeroEncoderUnits, first, second);
+	}
+
+	public void updateDegreesToEncoderUnitsRatio(double ratio){
+		degreesToEncoderUnitsRatio = ratio;
+	}
+
+	public void setAntigravConstant(double k) {
+		antiGravK = k;
+	}
+
+	public void setNewSetpoint(double setPoint){
```

Comment:
what units is `setPoint` in?

Reply:
encoder units. I'll add a comment to let others know that as well.